image-sync/oc-mirror/main.go

```diff
@@ -0,0 +1,62 @@
+package main
```

Comment:
Nit: I not have used golang for this, but plain `.sh`, since it's available as well. :man_shrugging:

Reply:
What about an alternative approach:

1. Use tempaltize to generate the configuration
2. Encode it base64, pass it in as environment variable
3. Decode and write it to a file

Reply:
i like the base64 option. i'll go for it